pyodbc.cpython-35m-x86_64-linux-gnu.so,sha256=R-dCKPNnWY5ffUYbBmBcB7JLTrhvuq0zFZX4fVoe994,527056
pyodbc-4.0.22.dist-info/DESCRIPTION.rst,sha256=zvBd0s-GAWA7NMmwHWkdPbRGltkhjHUiLpGLzUYkVWk,154
pyodbc-4.0.22.dist-info/METADATA,sha256=8wn-bzgijvHOHJ1pxCakh5pgwLpHzvZJC-2XCJ6W1kE,1082
pyodbc-4.0.22.dist-info/RECORD,,
pyodbc-4.0.22.dist-info/WHEEL,sha256=W-4s40OoP4gdldcaQ9BX8jWsxh8jVrPzLp2xl2ekMtI,104
pyodbc-4.0.22.dist-info/metadata.json,sha256=ufNqre7bfgXifQNIIt1rIqlqhikZp0mDMsRlnEwpJLY,1010
pyodbc-4.0.22.dist-info/top_level.txt,sha256=30jvYcCul6_wA26OT5c068GAHzqQLpd9WfQ0mSSsDTU,7
pyodbc-4.0.22.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
